src/main/java/step2/domain/LottoResult.java

```diff
@@ -0,0 +1,81 @@
+package step2.domain;
+
+import step2.domain.number.Count;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.HashMap;
+import java.util.Map;
+
+public class LottoResult {
+  private final Map<Count, Count> results;
+  private final Map<Count, Cash> outputInfo;
+  private Double yield;
+
+  private static final int MATCHING_THREE = 5000;
+  private static final int MATCHING_FOUR = 50000;
+  private static final int MATCHING_FIVE = 1500000;
+  private static final int MATCHING_SIX = 2000000000;
```

Comment:
돈과 같은 1000단위로 구분은 다음과 같이 underbar(_)를 활용하면 코드의 가독성이 좋아질 수 있어요!
```suggestion
  private static final int MATCHING_SIX = 2_000_000_000;
```